.eslintrc.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
     'linebreak-style': 0,
     'max-len': 0,
     'react/jsx-filename-extension': [2, { 'extensions': ['.js', '.jsx', '.ts', '.tsx'] }],
+    'no-console': 'off'
```

Comment:
does this mean any console logs will throw an error?

Reply:
Actually the opposite.  Any console log used to throw an error, so we had lots "ignore this console log" throughout the code.  This allows us to use console log without any errors.